common/config/beachball/changelog-custom-renders.ts

```diff
@@ -1,9 +1,13 @@
-import { PackageChangelogRenderInfo, ChangelogEntry } from 'beachball';
+import { ChangelogEntry, PackageChangelogRenderInfo } from 'beachball';
 import { getPrNumber, repoDetails } from './github-functions';
 
 const repoUrl = `https://github.com/${repoDetails.owner}/${repoDetails.repo}`;
+let features: ChangelogEntry[] = [];
```

Comment:
Does these need to be mutable global variables? Can they be moved inside the `renderPackageChangelog` fn

Reply:
Ah this is a good catch. Will update it now.